src/vm/nativeeventsource.cpp

```diff
@@ -0,0 +1,40 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+//
+// File: nativeeventsource.cpp
+// Abstract: This module implements native part of Event Source support in VM
+//
+
+//
```

Comment:
Since you have created a dedicated .cpp for this, you should create a dedicated header file as well.